src/API/Helpers/ApiClient.php

```diff
@@ -4,15 +4,39 @@
 use Auth0\SDK\API\Header\ContentType;
 use Auth0\SDK\API\Header\Telemetry;
 
+/**
+ * Class ApiClient
+ *
+ * @package Auth0\SDK\API\Helpers
+ */
 class ApiClient
 {
 
     const API_VERSION = '5.6.0';
 
+    /**
+     * Flag to turn telemetry headers off.
+     * Adjusted with self::disableInfoHeaders().
+     *
+     * @var boolean
+     */
     protected static $infoHeadersDataEnabled = true;
 
+    /**
+     * Current telemetry headers.
+     *
+     * @var InformationHeaders
+     */
     protected static $infoHeadersData;
 
+    /**
+     * Set new telemetry headers to be used for API requests.
+     * Used by dependents to report their package.
+     *
+     * @param InformationHeaders $infoHeadersData Object representing telemetry to send.
+     *
+     * @return null|void
```

Comment:
Why two possibilities? Line 43 could be adjusted to `return;` to always use `void`, which then implies a missing return type.